modular_pariah/modules/skrell/code/modules/client/preferences/species/skrell.dm

```diff
@@ -0,0 +1,39 @@
+/proc/generate_skrell_side_shots(list/sprite_accessories, key, list/sides)
```

Comment:
Why have this render from the side? Wouldn't it make more sense to be from the front instead?

Reply:
Lizards also have sideshots with their snouts, this seemed like a good idea for consistency reasons